website/animeketchup/ak_api/views.py

```diff
@@ -49,3 +51,59 @@ def landing_page(request):
 
 def anime_list(request):
     return render(request, 'anime_list.html')
+
```

Comment:
This is the poached function from watch_schedule_builder.py to process the calculation of the episodes and how long it'll take to watch the specified amount of episodes. More arguments can be passed to it going forward.